config/boards/station-p1.csc

```diff
@@ -1,6 +1,6 @@
 # Rockchip RK3399 hexa core 4GB LPDDR4 SoC eMMC GBE USB3
 BOARD_NAME="Station P1"
-BOARDFAMILY="rk3399"
+BOARDFAMILY="rockchip64"
```

Comment:
BOARDFAMILY="station"

Kernel configuration must be different or what?

Reply:
> Kernel configuration must be different or what?

Do you mean the entire "group" of stations (M1 and P1)?